utils/db.ts

```diff
@@ -2,10 +2,18 @@ import { Client } from 'pg';
 import type {cache, storeNarInfo} from "./types.d/dbTypes.ts";
 import type {CacheInfo, narUploadSuccessRequestBody} from "./types.d/apiTypes.ts";
 export default class Database {
-    private db:Client = new Client({
-        connectionString: process.env.POSTGRES_CONNECTION_STRING,
-    })
-    constructor(skipConnection:boolean = false){
+    
+    private db:Client
```

Comment:
I'd remove the Client construction from the constructor and I'd write it like this:
```suggestion
    private db:Client = new Client({
          user: process.env.POSTGRES_USER
          password: process.env.POSTGRES_PASSWORD
          host: process.env.POSTGRES_HOST
          port: process.env.POSTGRES_PORT
          database: "cache"
    })
```
Because this would break every single route file if we'd do it like this as the constructor is called without any arguments in the route files.